projects/saturn/src/main/java/io/fairspace/saturn/controller/ViewController.java

```diff
@@ -0,0 +1,58 @@
+package io.fairspace.saturn.controller;
+
+import jakarta.validation.Valid;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import io.fairspace.saturn.config.Services;
+import io.fairspace.saturn.controller.dto.CountDto;
+import io.fairspace.saturn.controller.dto.FacetsDto;
+import io.fairspace.saturn.controller.dto.ViewPageDto;
+import io.fairspace.saturn.controller.dto.ViewsDto;
+import io.fairspace.saturn.controller.dto.request.CountRequest;
+import io.fairspace.saturn.controller.dto.request.ViewRequest;
+
+@RestController
+@RequestMapping("${application.basePath}/views")
```

Comment:
For some controllers you have "/api/", for some "${application.basePath}", is there a reason for that?

Reply:
the only reason is my absent-mindedness :) 

I supposed to use ${application.basePath}, but probably it will never change and there parametrization is redundant. What do you think?

Reply:
but I made it as planned originally

Reply:
true, we could just have "/api" everywhere, not sure why it was made configurable, but let's keep the configuration option then

Reply:
pushed another approach setting it in one place at servlet level